client/app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.js

```diff
@@ -5,8 +5,25 @@ angular.module('managerApp').config(($stateProvider) => {
       'aliasInnerView@telecom.telephony.alias': {
         templateUrl: 'app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.html',
         controller: 'TelecomTelephonyAliasConfigurationCtrl',
-        controllerAs: 'AliasConfigurationCtrl',
+        controllerAs: '$ctrl',
       },
+      'featureView@telecom.telephony.alias.configuration': {
+        templateProvider: alias => `<div data-ng-include="::'app/telecom/telephony/alias/configuration/feature/${alias.featureType}/telecom-telephony-alias-configuration-${alias.featureType}.html'"></div>`,
+        controllerProvider: (alias) => {
+          const isNotEmptyFeature = alias.featureType !== 'empty';
+          return isNotEmptyFeature ? `TelecomTelephonyAliasConfiguration${_.capitalize(alias.featureType)}Ctrl` : null;
+        },
+        controllerAs: '$ctrl',
+      },
+    },
+    resolve: {
+      alias: ($stateParams, voipService) => voipService
+        .fetchSingleService($stateParams.billingAccount, $stateParams.serviceName)
+        .then((alias) => {
+          const _alias = _.clone(alias);
```

Comment:
Even if the ESLint rule `no-underscore-dangle` is disabled, coud we please respect this rule?

Reply:
Ok for sure :)